rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -217,15 +217,33 @@ type ReturnTerms: <"Specifies the type of return of a performance payout.">
+type PerformancePayout extends PayoutBase: <"Contains the necessary specifications for all performance payouts, encompassing equity return, dividend, variance, volatility and correlation products. The Performance Payout is either straight return, else an aggregation of multiple returns, hence a required choice between returnTerms and multiple portfolioReturnTerms.">
     [metadata key]
 
     observationTerms ObservationTerms (0..1) <"Defines how and when a performance type option or performance type swap is to be observed.">
     valuationDates ValuationDates (1..1) <"Defines how and when a performance type option or performance type swap is to be valued, including both interim and final valuation.">
     paymentDates PaymentDates (1..1) <"Defines the payment date schedule, as defined by the parameters that are needed to specify it, either in a parametric way or by reference to another schedule of dates (e.g. the valuation dates).">
     underlier Product (0..1) <"Identifies the underlying product that is referenced for pricing of the applicable leg in a swap.  Referenced in the '2018 ISDA CDM Equity Confirmation for Security Equity Swap' as Security.">
     fxFeature FxFeature (0..*) <"Defines quanto or composite FX features that are included in the swap leg.">
-    returnTerms ReturnTerms (1..1) <"Specifies the type of return of a performance payout.">
+    returnTerms ReturnTerms (0..1) <"Specifies the type of return of a performance payout.">
+    portfolioReturnTerms PortfolioReturnTerms (0..*) <"Specifies an individual type of return of a Performance Payout, when such individual return is part of an aggregation of multiple similar returns, at Performance Payout level">
+    initialValuationPrice PriceSchedule (0..1) <"Specifies the net initial valuation price(s) of the underlier at Performance Payout level. This price can be expressed either as an actual amount/currency, as a determination method, or by reference to another value specified in the swap document.">
+    [metadata address "pointsTo"=PriceQuantity->price]
+    interimValuationPrice PriceSchedule (0..1) <"Specifies the net initial valuation price(s) of the underlier at Performance Payout level. This price can be expressed either as an actual amount/currency, as a determination method, or by reference to another value specified in the swap document.">
+    [metadata address "pointsTo"=PriceQuantity->price]
+    finalValuationPrice PriceSchedule (0..1) <"Specifies the net final valuation price(s) of the underlier at Performance Payout level. This price can be expressed either as an actual amount/currency, as a determination method, or by reference to another value specified in the swap document.">
+    [metadata address "pointsTo"=PriceQuantity->price]
+
+    condition PorfolioOrStraightReturn:
```

Comment:
Add descriptions to conditions. It should address the "why we have this condition?" question. Maybe the last sentence you added in the `PerformancePayout` description belongs here?